nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server/src/main/java/org/apache/nifi/distributed/cache/server/DistributedCacheServer.java

```diff
@@ -68,6 +68,13 @@
         .required(false)
         .addValidator(StandardValidators.createDirectoryExistsValidator(true, true))
         .build();
+    public static final PropertyDescriptor MAX_READ_SIZE = new PropertyDescriptor.Builder()
+        .name("Maximum Read Size")
+        .description("The maximum number of network bytes to read for a single cache item")
+        .required(false)
+        .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
+        .defaultValue("1000000")
```

Comment:
Instead of using a positive integer, the `DATA_SIZE_VALIDATOR` supports standard formatting for data sizes:
```suggestion
        .addValidator(StandardValidators.DATA_SIZE_VALIDATOR)
        .defaultValue("1 MB")
```